src/annotation_processor.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: guide
+title: PDL Schema
+permalink: /annotation_processor
+excerpt: Documentation of using annotation processor in Rest.Li
+---
+
+# Annotation Processor
+This sections introduces a generic schema annotation processing tool in Rest.Li. Before reading this, it is recommended to be familiar with the concept of PathSpec in Rest.Li. [Insert link here]
+
+## Annotating Rest.Li Schemas
```

Comment:
Should we use "pegasus schemas" instead of Rest.Li?
Pegasus schemas can be used without rest.li framework and the annotation can still be applied

Reply:
I have changed. Meanwhile I really think this is subtle difference and reader might get confused by the term "Pegasus"